R/modelFitting.R

```diff
@@ -148,6 +148,7 @@ compBayes <- function(dat,
     if(sumPtm > 0){oneDat <- oneDat[which(oneDat$ptm == 0), ] }
   }
 
+  N_ <- nrow(oneDat)
```

Comment:
`N_` needs to be after the data has been subset in line 148 otherwise the number of expected `N_` is wrong when fitting the first model.

Reply:
Nice find! Would you mind mentioning this in a comment in the code?

Reply:
This sounds true.  Are you sure that nothing in the code required N_ prior to the new assignment?